src/chapters/strings/exercises.rst

```diff
@@ -51,11 +57,11 @@ Exercises: Strings
 
    `Code it at repl.it <https://repl.it/@launchcode/DNA-strings/>`__
 
-#. If we want to turn the string ``'JavaScript'`` into ``'JS'``, we might try
+3. PART A: If we want to turn the string ``'JavaScript'`` into ``'JS'``, we might try
    ``.remove()``. Unfortunately, there is no such method in JavaScript.
    However, we can use our cleverness to achieve the same result.
 
-   #. Use string concatenation and two ``slice()``'s to print ``'JS'`` from
+   #. Use string concatenation and two ``slice()``s to print ``'JS'`` from
```

Comment:
Small rephrasing and tick corrections: "Use string concatenation and two ``slice()`` methods to print ``'JS'`` from ``JavaScript``.